ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -300,6 +300,15 @@ echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_INSTALL_START"
 openshift-install --dir="${dir}" create manifests &
 wait "$!"
 
+# aws-outpost code - begin
+if [[ "${CLUSTER_TYPE}" == 'aws' ]] && [[ -f "${SHARED_DIR}/outpost_stack" ]]; then
+  ls -l "${SHARED_DIR}"
+  cat "${SHARED_DIR}/install-config.yaml"
+  sed -i "s/$(cat $SHARED_DIR/prv_subn)/$(cat $SHARED_DIR/outpost_prv_subn)/" $dir/openshift/99_openshift-cluster-api_worker-machineset-0.yaml
+  cp $SHARED_DIR/cluster-network-03-config.yml $dir/manifests/
```

Comment:
rename `cluster-network-03-config.yml` to `manifest_cluster-network-03-config.yml`, it will copy to `manifests` directly [1]

[1] https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L315-L324

Reply:
what's wrong with the copy command?